plugins/modules/pnp_intent.py

```diff
@@ -848,6 +974,17 @@ def get_want(self, config):
             'template_name': config.get('template_name')
         }
 
+        if self.want["pnp_type"] == "CatalystWLC":
+            self.want["staticIP"] = config.get('staticIP')
+            self.want["subnetMask"] = config.get('subnetMask')
+            self.want["gateway"] = config.get('gateway')
+            self.want["vlanId"] = config.get('vlanId')
+            self.want["ipInterfaceName"] = config.get('ipInterfaceName')
+
+        if self.want["pnp_type"] == "AccessPoint":
```

Comment:
pnp_type could be either CatalystWLC or Accesspoint. So we can do if and elif... Otherwise, we compare two times unnecessarily